conduct/faq.md

```diff
@@ -0,0 +1,77 @@
+
+We think the Jupyter community is awesome. If you're familiar with the Jupyter
+community, you'll probably notice that the Code basically matches what we
+already do. Think of this as documentation: we're taking implicit expectations
+about behavior and making them explicit.
+
+We're doing this because the Jupyter community is growing faster than any of us
+could have anticipated. This is on balance a very positive thing, but as we've
+grown past the point where it's possible to know the whole community, we think
+it's very important to be clear about our values.
+
+We know that the Jupyter community is open, friendly, and welcoming. We want to
+make sure everyone else knows it too.
+
+### What does it mean to "adopt" a Code of Conduct?
+
+For the most part, we don't think it means large changes. We think that the text
+does a really good job describing the way the Jupyter community already conducts
+itself. We expect that most people will simply continue to behave as they have
+in the past.
+
+However, we do expect that people will abide by the spirit and words of the Code
+of Conduct when in "official" Jupyter spaces. This code has been adopted by both
+the Jupyter Steering Council and JupyterDay/JupyterCon events. That means that
+it will apply both in community spaces and at our events.
+
+In practice, Jupyter spaces include mailing lists, various communication
+channels, and events.
+
+### What happens if someone violates the Code of Conduct?
+
+We are all stewards of our community, and are encouraged to participate in ways
+that defend the values highlighted in this document and help others understand
+when their actions go against these values (by engaging them and directing them
+to this document if necessary). If that doesn't work, or if you need more help,
+you can contact conduct@jupyter.org. For more details please see our [Reporting
+Guidelines](*reporting_online.md*).
+
+### What about events funded by Project Jupyter?
+
+This Code of Conduct will also cover any events that Project Jupyter funds.
+
+The project requires that any events Project Jupyter funds have policies and
+procedures in place for handling harassment. It's especially important to us
+that real-life events take steps to protect the physical and mental security of
+their participants.
+
+Event organizers will enforce the Code of Conduct.
+
+### Why do we need a Code of Conduct? Everyone knows not to be a jerk.
+
+Sadly, not everyone knows this.
+
+However, even if everyone was kind, everyone was compassionate, and everyone was
+familiar with codes of conduct it would still be incumbent upon our community to
+publish our own. Maintaining a Code of Conduct forces us to consider and
+articulate what kind of community we want to be, and serves as a constant
+reminder to put our best foot forward. But most importantly, it serves as a
+signpost to people looking to join our community that we feel these values are
+important.
+
```

Comment:
@carreau @fperez Would you find this to be a helpful addition?

```
### English is not my native language. I'm concerned that I may say or write words that when translated
        may inadvertently offend someone and prompt a Code of Conduct report. How can the committee help?

We understand that individuals come from different cultural backgrounds and native languages. We
empathize that complex discussion in a foreign language can be intimidating, and we wish to encourage
diversity across nationalities and cultures.

In the case of a report for verbal or written communication that does not include violation of physical
boundaries or personal safety, the committee will strongly consider the high likelihood the verbal or
written miscommunication is due to language translation instead of malicious intent. We expect that
individuals also consider this likelihood before filing a report.

```